.azure-pipelines/markers-check.yml

```diff
@@ -0,0 +1,12 @@
+steps:
+- script: |
+    set -x
+
+    pip3 install natsort
+
+    python3 ./.azure-pipelines/markers_check/markers_check.py tests
+    if [[ $? -ne 0 ]]; then
```

Comment:
is that necessary since the python code is using sys.exit()

Reply:
Here, we need to get the return value of python script and fail the pipeline. They are two things.

Reply:
Wouldn't sys.exit(1) fail pipeline?

Reply:
This only fail the python script. We need to get the return code of python script to fail the pipeline